noir-projects/aztec-nr/aztec/src/messages/processing/pending_tagged_log.nr

```diff
@@ -1,13 +1,9 @@
 use super::message_context::MessageContext;
-use protocol_types::{
-    address::AztecAddress,
-    constants::PRIVATE_LOG_SIZE_IN_FIELDS,
-    traits::{Deserialize, Serialize},
-};
+use protocol_types::{constants::PRIVATE_LOG_SIZE_IN_FIELDS, traits::Deserialize};
 
 /// A private log that was tagged for `recipient` and has not yet been processed, plus information related to the
 /// transaction in which it was emitted.
-#[derive(Deserialize, Serialize)]
+#[derive(Deserialize, Eq)]
```

Comment:
Serialize was unused here so I dropped it. Added `Eq` such that I can use it in tests.